JetNews/app/src/main/java/com/example/jetnews/ui/JetnewsApp.kt

```diff
@@ -125,28 +141,26 @@ private fun DrawerButton(
         colors.surface
     }
 
-    val surfaceModifier = modifier +
-            LayoutPadding(start = 8.dp, top = 8.dp, end = 8.dp, bottom = 0.dp) +
-            LayoutWidth.Fill
+    val surfaceModifier = modifier
+                .padding(start = 8.dp, top = 8.dp, end = 8.dp, bottom = 0.dp)
+                .fillMaxWidth()
     Surface(
         modifier = surfaceModifier,
         color = backgroundColor,
         shape = RoundedCornerShape(4.dp)
     ) {
-        TextButton(onClick = action, modifier = LayoutWidth.Fill) {
-            Row(arrangement = Arrangement.Start, modifier = LayoutWidth.Fill) {
-                VectorImage(
-                    modifier = LayoutGravity.Center,
-                    id = icon,
-                    tint = textIconColor
+        TextButton(onClick = action, modifier = Modifier.fillMaxWidth()) {
+            Row(arrangement = Arrangement.Start, modifier = Modifier.fillMaxWidth()) {
+                Image(
+                    asset = vectorResource(icon),
+                    colorFilter = ColorFilter(textIconColor, BlendMode.srcATop),
```

Comment:
I think this should be:

```suggestion
                    colorFilter = ColorFilter(textIconColor, BlendMode.srcIn),
```

Reply:
Yep! Works will use this for both.